### DISCUSSION TITLE
Authorise succeeds but just stops?

### DISCUSSION THREAD

**Discussion by richiema32** [DIGITALCRIMINALS/OnlyFans#366] [Q&A]
No authorisation issues but once it succeeds authorising, it just stops?
![image](https://user-images.githubusercontent.com/83547038/156274505-1e39e07e-ba2a-4642-b549-a73fb2908a4f.png)

Stays on this screen and doesn't move anywhere. Has previously worked literally 3-4 days ago.

**Comment by DIGITALCRIMINAL** [comment]
You probably have 10 trillion subscriptions and they all need to load in
